scripts/auth-open.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import json
+import os
+import base64
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+#
+# From : https://github.com/dcos/dcos/blob/master/test_util/helpers.py#L35
```

Comment:
Don't copy paste this. You are even taking the trouble to reference the copy/paste location; just import instead